packages/use-contractkit/package.json

```diff
@@ -32,6 +32,7 @@
     "@ethersproject/providers": "^5.3.0",
     "@ledgerhq/hw-transport-webusb": "^5.43.0",
     "@types/isomorphic-fetch": "^0.0.35",
+    "@walletconnect/registry": "git+https://github.com/WalletConnect/walletconnect-registry.git#production",
```

Comment:
it seems to be building properly https://use-contractkit-aemgs6fbu-c-labs.vercel.app/

Reply:
I'm sorry 🤦 I said 'block' but I meant 'bloat'. As in increase the bundle size significantly

Reply:
We only import a json file out of this repo, shouldn't impact the bundle size. It will however impact the install size, but I don't believe that's a big issue.